x/staking/keeper/slash.go

```diff
@@ -30,7 +30,7 @@ import (
 //
 //	Infraction was committed at the current height or at a past height,
 //	not at a height in the future
-func (k Keeper) Slash(ctx sdk.Context, consAddr sdk.ConsAddress, infractionHeight int64, power int64, slashFactor sdk.Dec) math.Int {
+func (k Keeper) Slash(ctx sdk.Context, consAddr sdk.ConsAddress, infractionHeight int64, power int64, slashFactor sdk.Dec, _ types.Infraction) math.Int {
```

Comment:
nit: can we add a small comment on why the infraction is ignored here 🙏  A link to a discussion would also be helpful

Reply:
This Slash implementation doesn't require the infraction to work but the IS one does. So it's just here to satisfy the StakingKeeper contract.

Reply:
Added a godoc.